t/parse/parens.xml

```diff
@@ -537,21 +537,21 @@
         </Math>
       </equation>
       <equation xml:id="S5.Ex21">
-        <Math mode="display" tex="\left|{a\over b}\right|" text="absolute-value@(a / b)" xml:id="S5.Ex21.m1">
+        <Math mode="display" tex="\left{\delimiter 69632778}{a\over b}\right{\delimiter 86414091}" text="delimited-⟨⟩@(a / b)" xml:id="S5.Ex21.m1">
```

Comment:
Hm, we may want to avoid the `{` and `}` entirely when reverting delimiters. The tex attribute on the Math above encounters an error in pdftex. What succeeded was the plainer form from parens.tex:
```tex
$\left\delimiter 69632778 {a\over b}\right \delimiter 86414091$
\bye
```

Since we have recently fixed a report about `tex` attribute reuse with TeX, I thought I would bring this up.

Reply:
Yeah, good catch. patched..